numba/targets/arrayobj.py

```diff
@@ -294,45 +273,110 @@ def _getitem_array_generic(context, builder, return_type, aryty, ary,
     output_shapes = []
     output_strides = []
 
-    for ax, (indexval, idxty) in enumerate(zip(indices, index_types)):
+    ax = 0
+    for indexval, idxty in zip(indices, index_types):
+        if idxty is types.ellipsis:
+            # Fill up missing dimensions at the middle
+            n_missing = aryty.ndim - len(indices) + 1
```

Comment:
Hmm... Since it's not documented, I'm not sure there's any point in supporting at all. Apparently it's only there for compatibility with the old Numeric package...